docs/competition/team-support/helpdesk.md

```diff
@@ -22,12 +22,10 @@ Helpdesk is the teams' primary source of support. Volunteers are expected to adv
  * Permanent marker
  * Large post-it notes
  * Radio
- * Emergency contact numbers for the Competition Team Support Coordinator (Rich Barlow) and the Competition Event Coordinator (Rob Spanton)
- * Spare robot badge for teams to borrow to try for size (must be clearly marked and returned to helpdesk ASAP)
+ * Emergency contact number for the Health and Safety Coordinator
```

Comment:
This isn't being done anymore, as we're using radios. Probably better worded as "Emergency contact channel for..."

Reply:
I think the phone number as a back-up is a good idea, in case either helpdesk or Scarzy's radio vanishes/is out of battery/is off